web-api/migration-terraform/main/lambdas/migration-segments.js

```diff
@@ -162,10 +162,7 @@ exports.handler = async event => {
   );
 
   const ranMigrations = {
-    ...(await hasMigrationRan('0001-update-websockets-gsi1pk.js')),
-    ...(await hasMigrationRan('0002-session-scope.js')),
-    ...(await hasMigrationRan('bug-0039-notice-of-trial-date.js')),
-    ...(await hasMigrationRan('bug-0040-case-received-at.js')),
+    // example: ...(await hasMigrationRan('9999-example-migration.js')),
```

Comment:
I can't comment on the line I want to... I believe we can also remove 44-62?

Reply:
ah of course! :) ty